src/actions/event.action.js

```diff
@@ -0,0 +1,36 @@
+export const SET_EVENT = 'current-event/set_event';
```

Comment:
take a look at events reducer - all actions and action creators are stored there (in reducer file), it is done to follow https://github.com/erikras/ducks-modular-redux